django/publicmapping/redistricting/templates/viewplan.html

```diff
@@ -1219,7 +1219,7 @@ <h3>{% blocktrans %}You can select up to three statistics at once to display in
         $('#steps').tabs({ show: function(event, ui) {
             if ($(this).tabs('option', 'selected') == 3) {
                 exportIndexFile.init();
-                createTwitterButton();
+                typeof createTwitterButton !== "undefined" && createTwitterButton();
```

Comment:
I think it's more specific to ask if what you have is a function instead of asking if it isn't undefined. This is what I would do: 
```js
if (typeof createTwitterButton === 'function') {
    createTwitterButton();
}
```

Reply:
I agree that checking if the type is a function is better, but I think I lean slightly in favor of an expression instead of a statement since it's more constrained and thus a bit more "semantic": you have some conditions that must be met to do what you ultimately want to happen, the function call. The `if` statement could do anything in its body. However, `if` statements are more straight forward in that they are meant to show conditional logic, more so than the boolean operator bailout expression trick, nice as it may be once you're familiar with it. 

In any case, I'm fine with either; just thought I'd throw that out there :smile: 

/pedantic

Reply:
My suggestion is written that way because I lean towards an if statement favoring readability over brevity. I don't buy the argument that semantics should always win over code that is easy to read. 
I'm also fine with either, my main comment was about checking for something more specific.